less/variables.less

```diff
@@ -24,3 +24,41 @@
 @border-real: #dee2e6;
 
 // Mobile var
+@phone: ~"(max-width: 500px)";
+
+// other mixins created
+.largestFont(){
+   font-size: 3.5rem; 
+//    width: 100%;
+}
+
+.anchorPropsNav(){
+    // text-decoration: none;
+    // margin:0% -65% 0 90%;
+    // font-size: 12px;
+    // color: black;
+    // padding: 100% 0%;
+    //  width: 100%;
+     text-decoration: none;
+     font-size: 12px;
+     color: black;
+     text-align: center;
+}
+
+.buttonTag{
+    background-color: black;
+    color: white;
+    cursor: pointer;
+    width: 90px;
+    height: 32px;
+    font-size: 1.5rem;
+    border-radius: 4px;
```

Comment:
Great job on this mixin. Another cool thing to remember is that all of the Less properties are applied here too, including `nesting`, using `psuedo` selectors, `variables`, and classic media queries. So you could add all of your `btn` properties including the animation, and mobile break point right here in the mixin and just call it as a property of any button of your choice in the page.